src/pages/home/Home.tsx

```diff
@@ -81,7 +82,7 @@ const Home = () => {
 };
 
 const HomeGuests = () => (
-  <>
+  <Fade>
```

Comment:
injec `Fade` via `<Container tag={...} />`

Reply:
it wont work for here since there are two elements being returned

Reply:
👍